nix/shell.sh

```diff
@@ -62,18 +62,11 @@ function unit {
 }
 
 function integ {
-    if [ "$1" == "help" ]; then
-        echo "The following tests are not supported:"
-        echo "- renegotiate_apache"
-        echo "   This test requires apache to be running. See codebuild/bin/s2n_apache.sh"
-        echo "    for more info."
-        return
-    fi
+    apache2_start
     if [[ -z "$1" ]]; then
```

Comment:
Are we starting apache even for single integ tests that don't need it just for simplicity? It doesn't slow things down too much?

Reply:
I went back and forth on it...no, startup time isn't a problem, I think the argument against doing this is added complexity.